spring-cloud-consul-discovery/src/main/java/org/springframework/cloud/consul/serviceregistry/ConsulAutoServiceRegistration.java

```diff
@@ -82,6 +86,11 @@ protected void register() {
 		super.register();
 	}
 
+	@Override
+	protected boolean shouldRegisterManagement() {
```

Comment:
No longer needed.

Reply:
I can't remove this until the changes to `spring-cloud-commons` land in dependencies - it will break `ConsulAutoServiceRegistrationManagementDisabledServiceTests`